boards/xtensa/intel_s1000_crb/support/create_board_img.py

```diff
@@ -188,9 +188,8 @@ def main():
     ipc_load_fw(SRAM_SIZE, FLASH_PART_TABLE_OFFSET)
 
     # pad zeros until FLASH_PART_TABLE_OFFSET
-    num_zero_pad = (FLASH_PART_TABLE_OFFSET / 4) - len(flash_content)
-    for x in range(int(num_zero_pad)):
-        flash_content.append(0)
+    num_zero_pad = (FLASH_PART_TABLE_OFFSET // 4) - len(flash_content)
+    flash_content += num_zero_pad*[0]
```

Comment:
This violates PEP8 (spaced around binary ops).

Reply:
Also please use `num_zero_pad * [0]`, which should clarify what happens there.

Reply:
No such rule in PEP 8 (not a PEP 8 fanatic either though :). They have this example:

![Skärmbild från 2019-03-20 11-18-41](https://user-images.githubusercontent.com/154768/54676859-f04adf80-4b01-11e9-9d4b-5a4acfc97e59.png)

Most common style I've seen in all languages no space around `*` and `/`, and spaces around `+` and `-`. Doing this looks a bit ugly and inconsistent to me:

```
a = b*c + d
z = e * f
```

Reply:
Got this too, so they're probably not against no spaces `*`:

![Skärmbild från 2019-03-20 11-25-53](https://user-images.githubusercontent.com/154768/54677364-ff7e5d00-4b02-11e9-9e47-22429a0b8d27.png)

Reply:
> No such rule in PEP 8

LOL, indeed. On a very thorough reading of https://www.python.org/dev/peps/pep-0008/#other-recommendations , 2nd bullet point:

> Always surround these binary operators with a single space on either side: assignment (=), augmented assignment (+=, -= etc.), comparisons (==, <, >, !=, <>, <=, >=, in, not in, is, is not), Booleans (and, or, not).

they don't speak about *all* binary operators, just *some*. And that's the same people which limit line size to orthodox and counter-productive 79 chars.

Well, anyway: I'll remain -1 until this is fixed, in Zephyr Python code, we have consistency re: that. Thanks.

Reply:
Yeah, FWIW, I think the line length limit in Python should be around 90, with common-sense exceptions. 79 is juust a bit too tight in many cases.

So is the Zephyr style to always put spaces around all binary operators, and write things like `a * b + c * d`? I could change it to that, though it feels a bit unusual to me.

Reply:
> So is the Zephyr style to always put spaces around all binary operators, and write things like a * b + c * d?

I personally never saw any other codestyle in projects which care about a codestyle at all. It is definitely the codestyle for Zephyr C code. And (most of?) the Python, Perl code is like that either.

Reply:
`a*b + c*d` is a lot easier to read for sure, but alright, changed.

Suspect it's some kind of overreaction to "these guys are writing impenetrable walls of code with zero whitespace", that turned into "let's just make them put spaces everywhere and that'll avoid that at least".

Reply:
> a*b + c*d is a lot easier to read for sure

Well, perhaps, but unlikely the same can be said for `foobarlong*another + longbarfoo*anotherlongvar`. That's the whole idea - to have simple, consistent, scalable rules which are easy to follow.

As for why PEP8 doesn't very explicitly forbid, my bet is that it's to cater for "scientist" users who indeed may love to write `a*b + c*d`. But that's interesting crowd which also loves to index arrays starting from arbitrary numbers, their recent language has stuff like `2x` (but not `x2`, in a sense that it has completely different meaning), etc. So, I'd take it as an exception, not as something to take for granted.

Thanks for changes!